mido/midifiles/midifiles.py

```diff
@@ -334,8 +345,8 @@ def _load(self, infile):
                 if self.debug:
                     _dbg('Track {}:'.format(i))
 
-                self.tracks.append(read_track(infile, debug=self.debug))
-                # Todo: used to ignore EOFError. I hope things still work.
+                self.tracks.append(read_track(infile, debug=self.debug, clip=self.clip))
```

Comment:
Fixed it! That's the one thing I have my linter turned off on, thanks for catching it :)